functions/Test-DbaComputerCertificateExpiration.ps1

```diff
@@ -0,0 +1,115 @@
+function Test-DbaComputerCertificateExpiration {
+    <#
+    .SYNOPSIS
+        Tests for certificates that are expiring soon
+
+    .DESCRIPTION
+        Tests for certificates that are expiring soon
+
+        By default, it tests candidates that are ideal for using with SQL Server's network encryption
+
+    .PARAMETER ComputerName
+        The target SQL Server instance or instances. Defaults to localhost. If target is a cluster, you must specify the distinct nodes.
+
+    .PARAMETER Credential
+        Allows you to login to $ComputerName using alternative credentials.
+
+    .PARAMETER Store
+        Certificate store - defaults to LocalMachine
+
+    .PARAMETER Folder
+        Certificate folder - defaults to My (Personal)
+
+    .PARAMETER Path
+        The path to a certificate - basically changes the path into a certificate object
+
+    .PARAMETER Type
+        The type of certificates to return. All, Service or SQL Server.
+
+        All is all certificates
+        Service is certificates that are candidates for SQL Server services (But may be for IIS, etc)
+        SQL Server is certificates currently in use by SQL Server
+
+    .PARAMETER Thumbprint
+        Return certificate based on thumbprint
+
+    .PARAMETER Threshold
+        Number of days before expiration to warn
```

Comment:
You should tell the user the default value here.

Reply:
good call!